.env

```diff
@@ -0,0 +1 @@
+REACT_APP_JWT_SECRET=something
```

Comment:
you should not push your .env file on github

Reply:
Ah yes, this was a workaround. I will learn how to use .env with Netlify and implement this in near future.